hack/test-update-storage-objects.sh

```diff
@@ -110,7 +110,7 @@ examples/persistent-volume-provisioning/rbd/rbd-storage-class.yaml,storageclasse
 )
 
 KUBE_OLD_API_VERSION="networking.k8s.io/v1,storage.k8s.io/v1beta1,extensions/v1beta1"
-KUBE_NEW_API_VERSION="networking.k8s.io/v1,storage.k8s.io/v1,extensions/v1beta1"
+KUBE_NEW_API_VERSION="networking.k8s.io/v1,storage.k8s.io/v1,extensions/v1beta1,policy/v1beta1"
```

Comment:
Was this just forgotten before? It surprises me to see any externally observable changes.

Reply:
Hm.. I don't know why this script has been failing. I'll try to look at it again later.

Reply:
I agree with @sttts, I wouldn't have expected this change as part of this PR...

Reply:
Looks like I did something wrong with storage. Other idea is that we couldn't move internal types without also moving registry to policy type.

Any ideas/pointers are welcome!

Reply:
@liggitt I'd like to move this PR forward.

`hack/test-update-storage-objects.sh` uses `KUBE_API_VERSIONS` to activate only certain group. Because internal type has moved to another type, we have to enable it too. This issue appears because `KUBE_API_VERSIONS` "is only for testing" and I think that it's ok that we had to update this test script.

Looks like @sttts also agree with that :)

Jordan, what do you think? Perhaps, we may accept it as-is? Otherwise, what actions are needed to merge this PR? Thanks!

Reply:
accepting as-is is fine with me

Reply:
Still not sure we really understand the issue here. We did not change the storage GV, did we? The internal types are registered even with `KUBE_API_VERSIONS` without the policy group. Is this a cohabitation problem that we need both groups?